pkg/quarterdeck/auth.go

```diff
@@ -97,6 +104,12 @@ func (s *Server) Register(c *gin.Context) {
 		return
 	}
 
+	// Now that the user exists in the database, send them a verification email
+	if err = s.SendVerificationEmail(user); err != nil {
+		log.Warn().Err(err).Msg("could not send verification email")
+		// TODO: If we fail to send the email should we create a retry task?
```

Comment:
Can we create a follow-on story for this please? It can go in the Advanced Registration and Login epic.

Reply:
Created!